DOCSP-30756: add list search indexes to cursor page (#704)
mongodb/docs-node: source/fundamentals/crud/read-operations/cursor.txt

   :start-after: start fetchAll cursor example
   :end-before: end fetchAll cursor example

> Return Search Indexes of Collection

Reviewer: ```suggestion
List Search Indexes
```

> Return Search Indexes of Collection

Reviewer: I: I'm not sure that this content belongs on this page. This page is about different ways to access data from a cursor, not about methods that return cursors. I think that you could add a section instead to the Indexes fundamentals page that covers this new method and the existing `listIndexes()` method. 

S: Move this section to Indexes and add information also about using `listIndexes()`. On this page, the only change that should be made is adding this method the list at the top of the page under "The following functions directly return cursors:"

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> When using a v7.0 and later {+mdb-server+} cluster, you can use the new `listSearchIndex()

Reviewer: ```suggestion
When connecting to {+mdb-server+} version 7.0 or later, you can use the new `listSearchIndex()
```

> When using a v7.0 and later {+mdb-server+} cluster, you can use the new `listSearchIndex()

Reviewer: ```suggestion
When using a v7.0 and later {+mdb-server+} cluster, you can use the new `listSearchIndexes()
```

> When using a v7.0 and later {+mdb-server+} cluster, you can use the new `listSearchIndex()
> <https://mongodb.github.io/node-mongodb-native/Next/classes/Collection.html#listSearchIndexes>`__
> to return a cursor with all the search indexes for the current
> collection. ``listSearchIndex()`` has an optional string parameter, ``name``, to

Reviewer: ```suggestion
When using a v7.0 and later {+mdb-server+} cluster, you can use the `listSearchIndex()
<https://mongodb.github.io/node-mongodb-native/Next/classes/Collection.html#listSearchIndexes>`__ method
to return a cursor that contains the search indexes for a
collection. ``listSearchIndex()`` has an optional string parameter, ``name``, to
```

> collection. ``listSearchIndex()`` has an optional string parameter, ``name``, to

Reviewer: S: Don't start sentences with monospace text if possible. Introduce methods as The ... method or types as the ... type

> return only the indexes with matching index names. It also has an

Reviewer: ```suggestion
return only the indexes with matching index names. It also takes an
```

optional ``aggregateOptions`` parameter. For more information, visit the
`API Documentation 

> <https://mongodb.github.io/node-mongodb-native/Next/interfaces/AggregateOptions.html>`__. 

Reviewer: S link the API documentation for `aggregateOptions` from that word and omit this sentence
